test-data/unit/check-incremental.test

```diff
@@ -4378,3 +4378,14 @@ from d import k
 [file t.py.2]
 from d import k
 # dummy change
+
+[case testBazelFlagIgnoresFileChanges]
```

Comment:
Are there any other `--bazel` behaviors we can check in tests without too much pain?